pkg/quota/evaluator/core/pods.go

```diff
@@ -131,10 +132,19 @@ func (p *podEvaluator) GroupKind() schema.GroupKind {
 	return api.Kind("Pod")
 }
 
-// Handles returns true of the evaluator should handle the specified operation.
-func (p *podEvaluator) Handles(operation admission.Operation) bool {
-	// TODO: update this if/when pods support resizing resource requirements.
-	return admission.Create == operation
+// Handles returns true of the evaluator should handle the specified attributes.
+func (p *podEvaluator) Handles(a admission.Attributes) bool {
```

Comment:
i have a goal to enable the following:
https://github.com/kubernetes/kubernetes/pull/51370

where do you recommend i best layer in the referenced goal with the need to support initializers?